### ISSUE TITLE
Coil:Heating:Water:Baseboard capacity not reported

### ISSUE THREAD

**Issue by muh182** [NREL/OpenStudio#5120]
Issue overview
--------------
EnergyPlus isn’t showing the coil object in the sizing results. and…

OpenStudio doesn’t have a “applySizingValues” method for the ZoneHVACBaseboardConvectiveWater object. 

## Current Behavior
The capacity does not get reported. 

## Expected Behavior
The capacity of Coil:Heating:Water:Baseboard should be reported.

## Steps to Reproduce
1. Use the attached model
2. Run the model
3. Open the SQL or the EIO file.
4. Observe that the "Heating Design Capacity" is not reported.

[coil_heating_water_baseboard.osm.zip](https://github.com/NREL/OpenStudio/files/14737491/coil_heating_water_baseboard.osm.zip)

This is likely an EnergyPlus bug too.

## Possible Solution
Fix the method.

### Environment
- Mac 14.4 but also occurred on Windows too.
- Occurred on 3.6 and now on 3.7

### Context
Students had to model baseboards and hard size their models last semester but could not find the capacity.

**Comment by joseph-robertson** [comment]
The ZoneHVACBaseboardConvectiveWater object does not contain any autosizable fields, so I believe we wouldn't expect it to have a “applySizingValues” method.

**Comment by joseph-robertson** [comment]
You are right, there doesn't appear to be a reported "Design Size Heating Design Capacity" field for the ZoneHVAC:Baseboard:Convective:Water object:

![image](https://github.com/NREL/OpenStudio/assets/8516790/03d8f18b-09b5-4f63-9578-b000be5e59b1)

**Comment by muh182** [comment]
Thanks @joseph-robertson for cross listing it.
